components/ListItemSquare/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function ListItemSquare({ movies }) {
+  return (
+    <>
+      {movies.map((movie) => {
+        return (
+          <ListItem key={movie.id}>
+            <Image
+              // height={100}
+              // width={100}
+              layout="fill"
+              objectFit="cover"
+              alt="picture of movie poster"
+              src={movie.picture}
+            />
+          </ListItem>
+        );
+      })}
+    </>
+  );
+}
+
+const ListItem = styled.li`
+  padding: 0;
+  display: flex;
+  flex-direction: row;
```

Comment:
row is the default value of the flex-direction. Therefore, it is not necessary to set it.🥳

```suggestion

```